### ISSUE TITLE
a11y > add form field label inside nav dropdown

### ISSUE THREAD

**Issue by ryanschaefer41** [vigetlabs/EcoCart#65]
Add form field label inside "Select Category" dropdown. Label should be "Category"
![image](https://user-images.githubusercontent.com/23317410/89204164-a3936780-d583-11ea-910c-52f52d7b0a76.png)

**Comment by ryanschaefer41** [comment]
@willyd332 I don't see this one implemented. Still getting a blank form field w/o a placeholder.
